client/packages/core/src/index.ts

```diff
@@ -601,4 +606,6 @@ export {
   PresenceSlice,
   PresenceResponse,
   InstaQLQueryParams,
+  InstantQuery,
```

Comment:
nit: would call InstaQLQuery, to match the others

Reply:
I named it this way because it _doesn't_ match the others.  `InstaQLQuery*` are all internal helpers that aren't user-friendly, whereas `InstantQuery` and `InstantQueryResult` are meant for external consumption.  Look at the generic signatures, they're super different.  Open to changing the naming scheme, but these should be separate.